internal/build/gulpfile.ts

```diff
@@ -50,7 +51,26 @@ export default series(
   parallel(
     runTask('buildModules'),
     runTask('buildFullBundle'),
-    runTask('generateTypesDefinitions'),
+    series(
```

Comment:
A task can be separated without using the `series· function, or you can refer to buildThemeChalk for splitting

Reply:
The task has been extracted into a separate file.